pkg/grok/base.go

```diff
@@ -87,6 +87,7 @@ func NewBase() Host {
 	h.Must("ISO8601_TIMEZONE", `Z|[+-]%{HOUR}(?::?%{MINUTE})`)
 	h.Must("ISO8601_SECOND", `%{SECOND}|60`)
 	h.Must("TIMESTAMP_ISO8601", `%{YEAR}-%{MONTHNUM}-%{MONTHDAY}[T ]%{HOUR}:?%{MINUTE}(?::?%{SECOND})?%{ISO8601_TIMEZONE}?`)
+	h.Must("DATESTAMP_ISO8601", `%{YEAR}-%{MONTHNUM}-%{MONTHDAY}`)
```

Comment:
Might as well use this in TIMESTAMP and also get the benefit of a little test coverage:
```suggestion
	h.Must("DATESTAMP_ISO8601", `%{YEAR}-%{MONTHNUM}-%{MONTHDAY}`)
	h.Must("TIMESTAMP_ISO8601", `%{DATESTAMP_ISO8601}[T ]%{HOUR}:?%{MINUTE}(?::?%{SECOND})?%{ISO8601_TIMEZONE}?`)
```